src/pysdmx/util/handlers.py

```diff
@@ -0,0 +1,59 @@
+"""Handlers file provide functions to make the code more readable."""
+
+from typing import Any
+
+
+# import pandas as pd
+
+
+def split_unique_id(obj_: str) -> tuple[str, str, str]:
+    """Split unique id.
+
+    Args:
+        obj_: String with the name of the object.
+
+    Returns:
+        agencyID, id and version.
+    """
+    data = obj_.split(":", 1)
+    agencyID = data[0]
+    data = data[1].split("(", 1)
+    id = data[0]
+    version = data[1].split(")", 1)[0]
+
+    return agencyID, id, version
+
+
+def split_from_urn(obj_: str, split_id: bool = True) -> Any:
+    """Split from urn.
+
+    Args:
+        obj_: String with the name of the object.
+        split_id: True splits the id.
+
+    Returns:
+        full id.
+    """
+    full_id = obj_.split("=", 1)[1]
+    return split_unique_id(full_id)
+
+
+def add_list(element: Any) -> Any:
+    """Adds data contained in the xml Dataset into a list if it is possible.
+
+    Args:
+        element: The elements contained in the Dataset.
+
+    Returns:
+        The elements in a list.
+    """
+    if not isinstance(element, list):
+        element = [element]
+    return element
+
+
+# def drop_na_all(df: pd.DataFrame):
```

Comment:
Should we drop commented out code and add it back when we are ready?

Reply:
Sure, I will remove it